codice/ddf
# DDF-2333 Add a command for seeding the metacard and product caches

#### What does this PR do?
Adds a `catalog:seed` command that allows the user to seed the local metacard and product caches from the enterprise or from specific federated sources.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@bdeining 
@glenhein 
@rzwiefel 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith 
@coyotesqrl 

#### How should this be tested?
Set up two DDFs. Ingest some products on DDF-1. On DDF-2, add DDF-1 as a federated source.

Run the `catalog:seed` command on DDF-2 with different values for the `--cql`, `--product-limit`, and `--source` options and verify that the expected metacards and products are added to the local caches.

#### What are the relevant tickets?
[DDF-2333](https://codice.atlassian.net/browse/DDF-2333)

#### Checklist:
